core/src/test/java/software/amazon/awssdk/core/http/AmazonHttpClientTest.java

```diff
@@ -93,7 +93,6 @@ public void testRetryIoExceptionFromHandler() throws Exception {
         final IOException exception = new IOException("BOOM");
 
         HttpResponseHandler<?> mockHandler = mock(HttpResponseHandler.class);
-        when(mockHandler.needsConnectionLeftOpen()).thenReturn(false);
```

Comment:
Why is this no being removed?

Reply:
Mockito 2.x detects unused stubbings.  This line was flagged as an unused stubbing by the Mockito engine. see: https://static.javadoc.io/org.mockito/mockito-core/2.15.0/org/mockito/exceptions/misusing/UnnecessaryStubbingException.html